lte/gateway/python/magma/pipelined/ng_manager/session_state_manager_util.py

```diff
@@ -59,8 +59,8 @@ def pdr_create_rule_entry(pdr_entry) -> PDRRuleEntry:
         local_f_teid = pdr_entry.pdi.local_f_teid
 
     # Get UE IP address
-    if len(pdr_entry.pdi.ue_ip_adr):
-        ue_ip_addr = pdr_entry.pdi.ue_ip_adr
+    #if len(pdr_entry.pdi.ue_ip_adr):
```

Comment:
why is this commented out

Reply:
Previously ue_ip_addr as a string. So we were putting condition for length check. Now i have modified proper condition.